### DISCUSSION TITLE
If Sunshine has Auto Detect Gamepad on, will it auto detect my DualSense when bluetoothed in to Steam Deck?  And will it also recognize the Steam Deck as a 360 if there is no DualSense connected?

### DISCUSSION THREAD

**Discussion by mxp83** [LizardByte/community#353] [Sunshine]
As title states, if I am bluetoothed in to my Deck (docked mode) I use my dualsense controller.  If Sunshine is set to auto detect, will it be DS4 when my PS controller is connected to the deck, and will it be 360 if I just use the Steam Deck itself (undocked).  I am connected using both latest Moonlight/Sunshine.

**Comment by ReenigneArcher** [comment]
Try it and report your findings.

[a comment by an automated account is not shown]
